### DISCUSSION TITLE
Safety of installing a root certificate

### DISCUSSION THREAD

**Discussion by kvsankar** [VirtualDrivers/Virtual-Display-Driver#216] [General]
The installation process involves installing the self-signed certificate as a root certificate. What are its implications? If the private key is leaked, can a website sign its certificate with this root certificate and deceive a browser? Can the certificate be removed after installation? How do I remove it? If I remove it, would the driver continue to work?

**Comment by bud3699** [comment]
You can remove it after installation but you'll need to add it again once you need to update or re-install

**Reply by kvsankar** [reply]
Okay, thank you.

**Comment by itsmikethetech** [comment]
We now have a signed certificate. The old one can be completely removed. :)
